packages/query/src/block-processor.ts

```diff
@@ -138,7 +138,7 @@ export class BlockProcessor implements BlockProcessorInterface {
    * Sync local state to present. This method will
    * - identify current synced height (or `-1n` to represent a 'pre-genesis' state)
```

Comment:
issue: I have been having trouble wrapping my mind around what currentHeight of -1 meant. But finally understood when I saw this comment. Can we do something like this:

```
const PRE_GENESIS_SYNC_HEIGHT = -1n;
...
let currentHeight = (await this.indexedDb.getFullSyncHeight()) ?? PRE_GENESIS_SYNC_HEIGHT;
...
if (currentHeight === PRE_GENESIS_SYNC_HEIGHT) {
```

Reply:
that's more clear, changed